recursion.rb

```diff
@@ -1,14 +1,43 @@
-
+# RC: n! = n * (n-1)!
+# BC: 1! = 1
+# BC: 0! = 1
 def fact(n)
+  return 1 if n == 0
+  return n * fact(n-1)
+
 end
 
 def fib(n)
+  return 1 if n == 1 || n == 2
+  return fib(n-1) + fib(n-2)
 end
```

Comment:
Yes, it's not a huge deal, and your implementation passes all the tests. Just something to think about!